config/environments/test.rb

```diff
@@ -13,7 +13,8 @@
   # Show full error reports and enable caching
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = true
-  config.cache_store = :memory_store
+  config.cache_store = :dalli_store, '127.0.0.1:11211',
+                          { :namespace =>  'ao3-v1', :expires_in =>  0, :compress => true , :pool_size => 10 }
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Space found before comma.